libs/external-db-postgres/src/sql_filter_transformer.ts

```diff
@@ -218,9 +225,11 @@ class FilterParser {
             case include:
                 return 'IN'
         }
+
+        return 
     }
 
-    orderBy(sort) {
+    orderBy(sort: Sort[]) {
         if (!Array.isArray(sort) || !sort.every(isObject)) {
             return EmptySort
         }
```

Comment:
We talked about it, but I'll write here too,
`orderBy ` purpose is to gen either Sort[] or anything else
in case of Sort[] to something and in case of anything else do something else.
Due to that, the input type is not Sort[].